### PR TITLE
add pprof

### PR DESCRIPTION
**Pull request by xiaoliu10**
What this PR does:
add pprof config
Which issue(s) this PR fixes:
#12

### CODE REVIEW

**Comment by DogBaoBao** [comment]
> What this PR does:
> add pprof config
> Which issue(s) this PR fixes:
> #12

merge develop first.

File: configs/conf.yaml [hunk 1]
```diff
@@ -47,3 +47,6 @@ static_resources:
       type: "EDS"
       lb_policy: "RoundRobin"
       connect_timeout: "5s"
+  pprofConf:
+    enable: true
+    addr: "0.0.0.0:6060"
```
Comment (DogBaoBao) [inline configs/conf.yaml:52]: addr can use unified model by select.
like the follows:

```yaml
  pprofConf:
    enable: true
    addr: "pprof" 

address:
    - name: "pprof"
      address:
        socket_address:
          protocol_type: "HTTP"
          address: "0.0.0.0"
          port: 6060
```

File: go.sum [hunk 2]
```diff
@@ -54,7 +54,7 @@ github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRF
 github.com/aliyun/alibaba-cloud-sdk-go v1.61.18/go.mod h1:v8ESoHo4SyHmuB4b1tJqDHxfTGEciD+yhvOU/5s1Rfk=
 github.com/apache/dubbo-getty v1.3.10 h1:ys5mwjPdxG/KwkPjS6EI0RzQtU6p6FCPoKpaFEzpAL0=
 github.com/apache/dubbo-getty v1.3.10/go.mod h1:x6rraK01BL5C7jUM2fPl5KMkAxLVIx54ZB8/XEOik9Y=
-github.com/apache/dubbo-go v1.5.1 h1:hYktTWnMJdzwY0NkvSqJfOERkwFApZ3mH/tQBLVGO34=
+github.com/apache/dubbo-go v1.5.1 h1:9bcGOkek0+x+zqtvykKpuOwkoJfMef0zNxsyPbWOafY=
```
Comment (DogBaoBao) [inline go.sum:57]: don't use this 9bcGOkek0+x+zqtvykKpuOwkoJfMef0zNxsyPbWOafY=, because it is only run in my PC

File: pkg/model/pprof.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model
+
+type PprofConf struct {
+	Enable bool   `yaml:"enable" json:"enable" mapstructure:"enable" default: false`
+	Addr   string `yaml:"addr" json:"addr" mapstructure:"addr" default: "0.0.0.0:6060"`
```
Comment (DogBaoBao) [inline pkg/model/pprof.go:22]: like the yaml

File: pkg/proxy/proxy_start.go [hunk 4]
```diff
@@ -60,6 +61,10 @@ func (p *Proxy) Start() {
 		ls := ListenerService{Listener: &s}
 		go ls.Start()
 	}
+
+	if conf.GetPprof().Enable {
+		go http.ListenAndServe(conf.GetPprof().Addr, nil)
```
Comment (DogBaoBao) [inline pkg/proxy/proxy_start.go:66]: I think maybe pprof goruntine can stop by command, it's  dync, so need some api to control it.

File: pkg/model/pprof.go [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model
+
+type PprofConf struct {
+	Enable bool   `yaml:"enable" json:"enable" mapstructure:"enable" default: false`
```
Comment (zhangshen023) [inline pkg/model/pprof.go:21]: default value of struct tag should be `default:"false"`. 
u may see this warning by command 'go vet pprof.go'

File: pkg/proxy/proxy_start.go [hunk 6]
```diff
@@ -19,10 +19,15 @@ package proxy
...
 	"encoding/json"
+	"net/http"
+	_ "net/http/pprof"
+	"strconv"
 	"sync"
 )
 
 import (
+	_ "github.com/apache/dubbo-go/metadata/service/inmemory"
+	_ "github.com/apache/dubbo-go/metadata/service/remote"
```
Comment (cityiron) [inline pkg/proxy/proxy_start.go:30]: why proxy_start import the two package?

File: pkg/model/pprof.go [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	Addr   string `yaml:"addr" json:"addr" mapstructure:"addr"`
```
Comment (williamfeng323) [inline pkg/model/pprof.go:22]: why we don't put the address together with the pprof?

File: cmd/proxy/proxy.go [hunk 8]
```diff
@@ -24,7 +24,9 @@ import (
 )
 
 import (
+	_ "github.com/apache/dubbo-go/metadata/service/inmemory"
```
Comment (AlexStocks) [inline cmd/proxy/proxy.go:27]: move it to the 3rd import block.

File: pkg/model/pprof.go (hunk 5 above)
Comment (xiaoliu10) [reply pkg/model/pprof.go]: I'll change it

File: pkg/proxy/proxy_start.go (hunk 6 above)
Comment (xiaoliu10) [reply pkg/proxy/proxy_start.go]: it has been deteled

**Review by AlexStocks — APPROVED** [review]

**Review by cityiron — APPROVED** [review]

**Review by williamfeng323 — APPROVED** [review]

**Pull request merged**